src/schema/__tests__/ecommerce/approve_order_mutation.test.js

```diff
@@ -32,7 +32,9 @@ describe("Approve Order Mutation", () => {
             }
             ... on OrderWithMutationFailure {
               error {
-                description
+                type
```

Comment:
🙌 I think we could make this whole 
```
error {
  type
  description
  data
}
``` 
a fragment so we could define it in one place and reuse it but we can do that later.